backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/slider/internal/BpkRangeSliderImpl.kt

```diff
@@ -108,44 +117,72 @@ private fun SlideRangeLabel(
     interactionSource: MutableInteractionSource,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier = modifier
-            .semantics { stateDescription = label }
-            .padding(bottom = BpkSpacing.Xl),
-        verticalArrangement = Arrangement.Center,
-        horizontalAlignment = Alignment.CenterHorizontally,
-    ) {
-        BpkText(
-            text = label,
-            color = BpkTheme.colors.textPrimaryInverse,
-            style = BpkTheme.typography.label2,
-            textAlign = TextAlign.Center,
+    LabelLayout(modifier) {
+        Box(
             modifier = Modifier
                 .background(
                     color = BpkTheme.colors.coreAccent,
                     shape = FlareShape(
-                        borderRadius = BpkBorderRadius.Sm,
-                        flareHeight = BpkSpacing.Sm,
+                        borderRadius = BorderRadius,
+                        flareHeight = FlareHeight,
                         pointerDirection = BpkFlarePointerDirection.Down,
                     ),
                 )
-                .padding(top = BpkSpacing.Sm)
-                .padding(bottom = BpkSpacing.Md)
-                .padding(horizontal = BpkSpacing.Md)
-                .semantics { invisibleToUser() },
-
-        )
-        Spacer(
-            modifier = Modifier.height(BpkSpacing.Sm),
-        )
+                .padding(bottom = FlareHeight)
+                .height(BpkSpacing.Lg)
+                .padding(horizontal = BpkSpacing.Md),
+            contentAlignment = Alignment.Center,
+        ) {
+            BpkText(
```

Comment:
put this in a box to ensure we can centre align the text vertically and ensure we always centre it no matter the font scale